# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2014-05-15 09:08+0100\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"Language: \n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=CHARSET\n"
"Content-Transfer-Encoding: 8bit\n"

#: core/Rpg.py:53
msgid "ERROR_UNKNOWN_SELECTED_WORLD"
msgstr ""

#: core/Rpg.py:60
msgid "ERROR_NO_SELECTED_PLAYER"
msgstr ""

#: core/Rpg.py:84
msgid "PLAYER_SELECTION"
msgstr ""

#: core/Rpg.py:85
msgid "CHOICE_NEW_PLAYER"
msgstr ""

#: core/Rpg.py:86
msgid "CHOICE_EXISTING_PLAYER"
msgstr ""

#: core/Rpg.py:89
msgid "CHOICE_QUESTION"
msgstr ""

#: core/Rpg.py:110 core/commands/createPlayer.py:46
msgid "LOGIN_PROMPT"
msgstr ""

#: core/Rpg.py:113 core/commands/createPlayer.py:54
msgid "PASSWORD_PROMPT"
msgstr ""

#: core/Rpg.py:222
msgid "COMMAND_PROMPT"
msgstr ""

#: core/command_factory.py:39 core/commands/help.py:38
#: tests/commands/testHelp.py:26 tests/commands/testHelp.py:75
#: tests/commands/testLook.py:15 tests/commands/testLook.py:39
#: tests/commands/testLook.py:51 tests/commands/testLook.py:56
#: tests/commands/testLook.py:62 tests/commands/testLook.py:67
#: tests/commands/testLook.py:72 tests/commands/testLook.py:77
#: tests/commands/testLook.py:82 tests/commands/testLook.py:90
#: tests/commands/testLook.py:95 tests/commands/testLook.py:100
#: tests/commands/testLook.py:105 tests/commands/testLook.py:111
#: tests/commands/testLook.py:116 tests/commands/testLook.py:124
msgid "LOOK_COMMAND"
msgstr ""

#: core/command_factory.py:40 core/commands/help.py:22
#: tests/commands/testHelp.py:18 tests/commands/testHelp.py:59
#: tests/commands/testTalk.py:14 tests/commands/testTalk.py:19
#: tests/commands/testTalk.py:24 tests/commands/testTalk.py:29
#: tests/commands/testTalk.py:34 tests/commands/testTalk.py:39
#: tests/commands/testTalk.py:44 tests/commands/testTalk.py:49
#: tests/commands/testTalk.py:54
msgid "TALK_COMMAND"
msgstr ""

#: core/command_factory.py:41 core/commands/help.py:26
#: tests/commands/testHelp.py:20 tests/commands/testHelp.py:63
#: tests/commands/testMove.py:14 tests/commands/testMove.py:19
#: tests/commands/testMove.py:24 tests/commands/testMove.py:29
#: tests/commands/testMove.py:34 tests/commands/testMove.py:39
#: tests/commands/testMove.py:44 tests/commands/testMove.py:49
msgid "MOVE_COMMAND"
msgstr ""

#: core/command_factory.py:42 core/commands/help.py:30
#: tests/commands/testEnter.py:14 tests/commands/testEnter.py:19
#: tests/commands/testEnter.py:24 tests/commands/testEnter.py:29
#: tests/commands/testEnter.py:34 tests/commands/testEnter.py:39
#: tests/commands/testEnter.py:44 tests/commands/testEnter.py:46
#: tests/commands/testEnter.py:51 tests/commands/testEnter.py:53
#: tests/commands/testEnter.py:58 tests/commands/testEnter.py:63
#: tests/commands/testExit.py:53 tests/commands/testExit.py:60
#: tests/commands/testHelp.py:22 tests/commands/testHelp.py:67
msgid "ENTER_COMMAND"
msgstr ""

#: core/command_factory.py:43 core/commands/help.py:34
#: tests/commands/testExit.py:13 tests/commands/testExit.py:18
#: tests/commands/testExit.py:23 tests/commands/testExit.py:28
#: tests/commands/testExit.py:33 tests/commands/testExit.py:38
#: tests/commands/testExit.py:43 tests/commands/testExit.py:48
#: tests/commands/testExit.py:55 tests/commands/testExit.py:62
#: tests/commands/testHelp.py:24 tests/commands/testHelp.py:71
msgid "EXIT_COMMAND"
msgstr ""

#: core/command_factory.py:44 core/commands/help.py:51
#: tests/commands/testDrop.py:35 tests/commands/testDrop.py:42
#: tests/commands/testDrop.py:91 tests/commands/testDrop.py:98
#: tests/commands/testDrop.py:105 tests/commands/testDrop.py:112
#: tests/commands/testDrop.py:120 tests/commands/testDrop.py:127
#: tests/commands/testDrop.py:134 tests/commands/testDrop.py:141
#: tests/commands/testDrop.py:148 tests/commands/testDrop.py:158
#: tests/commands/testDrop.py:169 tests/commands/testDrop.py:176
#: tests/commands/testDrop.py:183 tests/commands/testDrop.py:190
#: tests/commands/testDrop.py:197 tests/commands/testDrop.py:204
#: tests/commands/testDrop.py:211 tests/commands/testDrop.py:218
#: tests/commands/testDrop.py:225 tests/commands/testDrop.py:232
#: tests/commands/testDrop.py:239 tests/commands/testDrop.py:246
#: tests/commands/testHelp.py:33 tests/commands/testHelp.py:87
#: tests/commands/testInventory.py:24 tests/commands/testInventory.py:32
#: tests/commands/testTake.py:15 tests/commands/testTake.py:20
#: tests/commands/testTake.py:25 tests/commands/testTake.py:30
#: tests/commands/testTake.py:35 tests/commands/testTake.py:40
#: tests/commands/testTake.py:46 tests/commands/testTake.py:51
#: tests/commands/testTake.py:56 tests/commands/testTake.py:61
#: tests/commands/testTake.py:66 tests/commands/testTake.py:71
#: tests/commands/testTake.py:76 tests/commands/testTake.py:81
#: tests/commands/testTake.py:87 tests/commands/testTake.py:92
#: tests/commands/testTake.py:97 tests/commands/testTake.py:102
#: tests/commands/testTake.py:108 tests/commands/testTake.py:113
#: tests/commands/testTake.py:118 tests/commands/testTake.py:123
#: tests/commands/testTake.py:128 tests/commands/testTake.py:131
#: tests/commands/testTake.py:136 tests/commands/testTake.py:138
#: tests/commands/testTake.py:141 tests/commands/testTake.py:147
#: tests/commands/testTake.py:152 tests/commands/testTake.py:157
#: tests/commands/testTake.py:162 tests/commands/testTake.py:167
#: tests/commands/testTake.py:172 tests/commands/testTake.py:177
#: tests/commands/testTake.py:182 tests/commands/testTake.py:187
#: tests/commands/testTake.py:192 tests/commands/testTake.py:197
#: tests/commands/testTake.py:202
msgid "TAKE_COMMAND"
msgstr ""

#: core/command_factory.py:45 core/commands/help.py:61
#: tests/commands/testDrop.py:15 tests/commands/testDrop.py:20
#: tests/commands/testDrop.py:25 tests/commands/testDrop.py:30
#: tests/commands/testDrop.py:37 tests/commands/testDrop.py:44
#: tests/commands/testDrop.py:50 tests/commands/testDrop.py:55
#: tests/commands/testDrop.py:60 tests/commands/testDrop.py:65
#: tests/commands/testDrop.py:70 tests/commands/testDrop.py:75
#: tests/commands/testDrop.py:80 tests/commands/testDrop.py:85
#: tests/commands/testDrop.py:93 tests/commands/testDrop.py:100
#: tests/commands/testDrop.py:107 tests/commands/testDrop.py:114
#: tests/commands/testDrop.py:122 tests/commands/testDrop.py:129
#: tests/commands/testDrop.py:136 tests/commands/testDrop.py:143
#: tests/commands/testDrop.py:150 tests/commands/testDrop.py:153
#: tests/commands/testDrop.py:160 tests/commands/testDrop.py:163
#: tests/commands/testDrop.py:171 tests/commands/testDrop.py:178
#: tests/commands/testDrop.py:185 tests/commands/testDrop.py:192
#: tests/commands/testDrop.py:199 tests/commands/testDrop.py:206
#: tests/commands/testDrop.py:213 tests/commands/testDrop.py:220
#: tests/commands/testDrop.py:227 tests/commands/testDrop.py:234
#: tests/commands/testDrop.py:241 tests/commands/testDrop.py:248
#: tests/commands/testHelp.py:41 tests/commands/testHelp.py:91
msgid "DROP_COMMAND"
msgstr ""

#: core/command_factory.py:46 core/commands/help.py:42
#: tests/commands/testHelp.py:28 tests/commands/testHelp.py:79
#: tests/commands/testInventory.py:14 tests/commands/testInventory.py:19
#: tests/commands/testInventory.py:26 tests/commands/testInventory.py:34
msgid "INVENTORY_COMMAND"
msgstr ""

#: core/command_factory.py:47 core/commands/help.py:42
#: tests/commands/testHelp.py:28 tests/commands/testHelp.py:79
msgid "INVENTORY_SHORT_COMMAND"
msgstr ""

#: core/command_factory.py:48 core/commands/help.py:70
#: tests/commands/testHelp.py:15 tests/commands/testHelp.py:48
#: tests/commands/testHelp.py:55 tests/commands/testHelp.py:95
msgid "HELP_COMMAND"
msgstr ""

#: core/command_factory.py:49 core/commands/help.py:46
#: tests/commands/testHelp.py:30 tests/commands/testHelp.py:83
#: tests/commands/testOpen.py:14 tests/commands/testOpen.py:19
#: tests/commands/testOpen.py:24 tests/commands/testOpen.py:29
#: tests/commands/testOpen.py:34 tests/commands/testOpen.py:39
#: tests/commands/testOpen.py:44 tests/commands/testOpen.py:49
#: tests/commands/testOpen.py:54 tests/commands/testOpen.py:63
#: tests/commands/testOpen.py:68 tests/commands/testOpen.py:72
#: tests/commands/testOpen.py:80 tests/commands/testOpen.py:84
#: tests/commands/testOpen.py:92 tests/commands/testOpen.py:101
#: tests/commands/testOpen.py:105 tests/commands/testOpen.py:109
#: tests/commands/testOpen.py:113 tests/commands/testOpen.py:117
msgid "OPEN_COMMAND"
msgstr ""

#: core/command_factory.py:75 core/commands/help.py:74
#: tests/commands/testHelp.py:50 tests/commands/testHelp.py:99
msgid "QUIT_COMMAND"
msgstr ""

#: core/command_factory.py:75 core/commands/help.py:74
#: tests/commands/testHelp.py:50 tests/commands/testHelp.py:99
msgid "QUIT_SHORT_COMMAND"
msgstr ""

#: core/command_factory.py:84 core/command_factory.py:97
msgid "ERROR_UNKNOWN_COMMAND"
msgstr ""

#: core/commands/createPlayer.py:23 tests/commands/testCreatePlayer.py:19
#: tests/commands/testCreatePlayer.py:24
msgid "ERROR_SIGNUP_NOT_ENOUGH_ARGUMENTS"
msgstr ""

#: core/commands/createPlayer.py:28 core/commands/createPlayer.py:49
#: tests/commands/testCreatePlayer.py:29 tests/commands/testCreatePlayer.py:34
msgid "ERROR_SIGNUP_LOGIN_ALREADY_USED"
msgstr ""

#: core/commands/createPlayer.py:32 tests/commands/testCreatePlayer.py:39
#: tests/commands/testCreatePlayer.py:44
msgid "ERROR_SIGNUP_INVALID_GENDER"
msgstr ""

#: core/commands/createPlayer.py:36 tests/commands/testCreatePlayer.py:49
#: tests/commands/testCreatePlayer.py:54
msgid "ERROR_SIGNUP_INVALID_SPECIES"
msgstr ""

#: core/commands/createPlayer.py:55
msgid "CONFIRM_PASSWORD_PROMPT"
msgstr ""

#: core/commands/createPlayer.py:58
msgid "ERROR_SIGNUP_PASSWORDS"
msgstr ""

#: core/commands/createPlayer.py:64
msgid "GENDER_SELECTION"
msgstr ""

#: core/commands/createPlayer.py:70
msgid "GENDER_PROMPT"
msgstr ""

#: core/commands/createPlayer.py:84
msgid "SPECIES_SELECTION"
msgstr ""

#: core/commands/createPlayer.py:91
msgid "SPECIES_PROMPT"
msgstr ""

#: core/commands/createPlayer.py:104 tests/commands/testCreatePlayer.py:59
msgid "PLAYER_CREATION_CONFIRMATION"
msgstr ""

#: core/commands/drop.py:20 tests/commands/testDrop.py:52
#: tests/commands/testDrop.py:57 tests/commands/testDrop.py:62
#: tests/commands/testDrop.py:67
msgid "ERROR_DROP_NO_ITEM_GIVEN"
msgstr ""

#: core/commands/drop.py:22 tests/commands/testDrop.py:27
#: tests/commands/testDrop.py:32
msgid "ERROR_DROP_TOO_LOW_QUANTITY"
msgstr ""

#: core/commands/drop.py:24 tests/commands/testDrop.py:17
#: tests/commands/testDrop.py:22
msgid "ERROR_DROP_INVALID_FORMAT_QUANTITY"
msgstr ""

#: core/commands/drop.py:26 tests/commands/testDrop.py:138
#: tests/commands/testDrop.py:145
msgid "ERROR_DROP_INVALID_CONTAINER_INDEX"
msgstr ""

#: core/commands/drop.py:40 tests/commands/testDrop.py:72
#: tests/commands/testDrop.py:77
msgid "ERROR_DROP_UNKNOWN_ITEM"
msgstr ""

#: core/commands/drop.py:45 tests/commands/testDrop.py:82
#: tests/commands/testDrop.py:87
msgid "ERROR_DROP_ITEM_NOT_AVAILABLE"
msgstr ""

#: core/commands/drop.py:47 tests/commands/testDrop.py:39
#: tests/commands/testDrop.py:46
#, python-format
msgid "ERROR_DROP_QUANTITY_TOO_HIGH_%s"
msgstr ""

#: core/commands/drop.py:62 tests/commands/testDrop.py:173
#: tests/commands/testDrop.py:187 tests/commands/testDrop.py:201
#: tests/commands/testDrop.py:215 tests/commands/testDrop.py:229
#: tests/commands/testDrop.py:243
#, python-format
msgid "DROP_CONFIRMATION_%(quantity)s_%(name)s"
msgstr ""

#: core/commands/enter.py:21 tests/commands/testEnter.py:16
#: tests/commands/testEnter.py:21
msgid "ERROR_ENTER_NO_PLACE_GIVEN"
msgstr ""

#: core/commands/enter.py:27 core/commands/exit.py:27
#: tests/commands/testEnter.py:26 tests/commands/testEnter.py:31
#: tests/commands/testEnter.py:48 tests/commands/testEnter.py:55
#: tests/commands/testExit.py:25 tests/commands/testExit.py:30
#: tests/commands/testExit.py:45 tests/commands/testExit.py:50
msgid "ERROR_ENTER_PLACE_NOT_AVAILABLE"
msgstr ""

#: core/commands/enter.py:32 tests/commands/testEnter.py:60
#: tests/commands/testEnter.py:65
msgid "ENTER_CONFIRMATION"
msgstr ""

#: core/commands/exit.py:21 tests/commands/testExit.py:15
#: tests/commands/testExit.py:20
msgid "ERROR_EXIT_NO_PLACE_GIVEN"
msgstr ""

#: core/commands/exit.py:30 tests/commands/testExit.py:57
#: tests/commands/testExit.py:64
msgid "EXIT_CONFIRMATION"
msgstr ""

#: core/commands/help.py:22 tests/commands/testHelp.py:18
#: tests/commands/testHelp.py:59
msgid "CHARACTER_TOKEN"
msgstr ""

#: core/commands/help.py:22 tests/commands/testHelp.py:18
#: tests/commands/testHelp.py:59
msgid "SENTENCE_TOKEN"
msgstr ""

#: core/commands/help.py:23 tests/commands/testHelp.py:19
#: tests/commands/testHelp.py:60
msgid "TALK_COMMAND_DESCRIPTION"
msgstr ""

#: core/commands/help.py:27 tests/commands/testHelp.py:21
#: tests/commands/testHelp.py:64
msgid "MOVE_COMMAND_DESCRIPTION"
msgstr ""

#: core/commands/help.py:31 tests/commands/testHelp.py:23
#: tests/commands/testHelp.py:68
msgid "ENTER_COMMAND_DESCRIPTION"
msgstr ""

#: core/commands/help.py:35 tests/commands/testHelp.py:25
#: tests/commands/testHelp.py:72
msgid "EXIT_COMMAND_DESCRIPTION"
msgstr ""

#: core/commands/help.py:39 tests/commands/testHelp.py:27
#: tests/commands/testHelp.py:76
msgid "LOOK_COMMAND_DESCRIPTION"
msgstr ""

#: core/commands/help.py:43 tests/commands/testHelp.py:29
#: tests/commands/testHelp.py:80
msgid "INVENTORY_COMMAND_DESCRIPTION"
msgstr ""

#: core/commands/help.py:46 core/commands/help.py:55 core/commands/help.py:65
#: tests/commands/testHelp.py:30 tests/commands/testHelp.py:37
#: tests/commands/testHelp.py:45 tests/commands/testHelp.py:83
#: tests/commands/testHelp.py:87 tests/commands/testHelp.py:91
msgid "CONTAINER_INDEX_TOKEN"
msgstr ""

#: core/commands/help.py:46 core/commands/help.py:54 core/commands/help.py:64
#: tests/commands/testHelp.py:30 tests/commands/testHelp.py:36
#: tests/commands/testHelp.py:44 tests/commands/testHelp.py:83
#: tests/commands/testHelp.py:87 tests/commands/testHelp.py:91
msgid "CONTAINER_TOKEN"
msgstr ""

#: core/commands/help.py:47 tests/commands/testHelp.py:31
#: tests/commands/testHelp.py:84
msgid "OPEN_COMMAND_DESCRIPTION"
msgstr ""

#: core/commands/help.py:52 core/commands/help.py:62
#: tests/commands/testHelp.py:34 tests/commands/testHelp.py:42
#: tests/commands/testHelp.py:87 tests/commands/testHelp.py:91
msgid "QUANTITY_TOKEN"
msgstr ""

#: core/commands/help.py:53 core/commands/help.py:63
#: tests/commands/testHelp.py:35 tests/commands/testHelp.py:43
#: tests/commands/testHelp.py:87 tests/commands/testHelp.py:91
msgid "ITEM_NAME_TOKEN"
msgstr ""

#: core/commands/help.py:57 tests/commands/testHelp.py:39
#: tests/commands/testHelp.py:88
msgid "TAKE_COMMAND_DESCRIPTION"
msgstr ""

#: core/commands/help.py:67 tests/commands/testHelp.py:47
#: tests/commands/testHelp.py:92
msgid "DROP_COMMAND_DESCRIPTION"
msgstr ""

#: core/commands/help.py:71 tests/commands/testHelp.py:49
#: tests/commands/testHelp.py:96
msgid "HELP_COMMAND_DESCRIPTION"
msgstr ""

#: core/commands/help.py:75 tests/commands/testHelp.py:51
#: tests/commands/testHelp.py:100
msgid "QUIT_COMMAND_DESCRIPTION"
msgstr ""

#: core/commands/help.py:80 tests/commands/testHelp.py:17
msgid "AVAILABLE_COMMANDS_TITLE"
msgstr ""

#: core/commands/item_interaction.py:50 tests/commands/testDrop.py:109
#: tests/commands/testDrop.py:116 tests/commands/testOpen.py:26
#: tests/commands/testOpen.py:31 tests/commands/testTake.py:99
#: tests/commands/testTake.py:104
msgid "ERROR_CONTAINER_NOT_AVAILABLE"
msgstr ""

#: core/commands/item_interaction.py:56 tests/commands/testDrop.py:152
#: tests/commands/testDrop.py:155 tests/commands/testDrop.py:162
#: tests/commands/testDrop.py:165 tests/commands/testOpen.py:70
#: tests/commands/testOpen.py:82 tests/commands/testOpen.py:103
#: tests/commands/testOpen.py:111 tests/commands/testTake.py:130
#: tests/commands/testTake.py:133 tests/commands/testTake.py:140
#: tests/commands/testTake.py:143
#, python-format
msgid "ERROR_OUT_OF_RANGE_ITEM_CONTAINER_INDEX_%d"
msgstr ""

#: core/commands/item_interaction.py:59 tests/commands/testDrop.py:124
#: tests/commands/testDrop.py:131 tests/commands/testOpen.py:46
#: tests/commands/testOpen.py:51 tests/commands/testTake.py:110
#: tests/commands/testTake.py:115
msgid "ERROR_MULTIPLE_CONTAINERS_AVAILABLE"
msgstr ""

#: core/commands/look.py:20 tests/commands/testLook.py:62
#: tests/commands/testLook.py:67
msgid "LOOK_REGION_PARAM"
msgstr ""

#: core/commands/look.py:21 tests/commands/testLook.py:72
#: tests/commands/testLook.py:77
msgid "LOOK_CHARACTERS_PARAM"
msgstr ""

#: core/commands/look.py:22 tests/commands/testLook.py:82
#: tests/commands/testLook.py:90
msgid "LOOK_DIRECTIONS_PARAM"
msgstr ""

#: core/commands/look.py:23 tests/commands/testLook.py:95
#: tests/commands/testLook.py:100
msgid "LOOK_PLACES_PARAM"
msgstr ""

#: core/commands/look.py:24 tests/commands/testLook.py:105
#: tests/commands/testLook.py:111
msgid "LOOK_OBJECTS_PARAM"
msgstr ""

#: core/commands/look.py:25 tests/commands/testLook.py:116
#: tests/commands/testLook.py:124
msgid "LOOK_CONTAINERS_PARAM"
msgstr ""

#: core/commands/look.py:37 tests/commands/testLook.py:53
#: tests/commands/testLook.py:58
msgid "ERROR_LOOK_UNKNOWN_SECTION"
msgstr ""

#: core/commands/look.py:96 tests/commands/testLook.py:18
#: tests/commands/testLook.py:64
#, python-format
msgid "CURRENT_REGION_%s"
msgstr ""

#: core/commands/look.py:100 tests/commands/testLook.py:20
#: tests/commands/testLook.py:74
msgid "PRESENT_CHARACTERS"
msgstr ""

#: core/commands/look.py:107 tests/commands/testLook.py:23
#: tests/commands/testLook.py:85
msgid "AVAILABLE_DIRECTIONS"
msgstr ""

#: core/commands/look.py:114 tests/commands/testLook.py:26
#: tests/commands/testLook.py:97
msgid "AVAILABLE_PLACES"
msgstr ""

#: core/commands/look.py:121 tests/commands/testLook.py:29
#: tests/commands/testLook.py:107
msgid "AVAILABLE_ITEMS"
msgstr ""

#: core/commands/look.py:128 tests/commands/testLook.py:32
#: tests/commands/testLook.py:118
msgid "AVAILABLE_ITEMS_CONTAINERS"
msgstr ""

#: core/commands/move.py:19 tests/commands/testMove.py:16
#: tests/commands/testMove.py:21
msgid "ERROR_MOVE_NO_DIRECTION_GIVEN"
msgstr ""

#: core/commands/move.py:23 tests/commands/testMove.py:26
#: tests/commands/testMove.py:31
#, python-format
msgid "ERROR_MOVE_INVALID_DIRECTION_%s"
msgstr ""

#: core/commands/move.py:30 tests/commands/testMove.py:36
#: tests/commands/testMove.py:41
msgid "ERROR_MOVE_DIRECTION_NOT_AVAILABLE"
msgstr ""

#: core/commands/move.py:37 tests/commands/testMove.py:46
#, python-format
msgid "MOVE_CONFIRMATION_%s"
msgstr ""

#: core/commands/open.py:18 tests/commands/testOpen.py:16
#: tests/commands/testOpen.py:21
msgid "ERROR_OPEN_NO_CONTAINER_PROVIDED"
msgstr ""

#: core/commands/open.py:44 tests/commands/testOpen.py:57
#: tests/commands/testOpen.py:75 tests/commands/testOpen.py:87
#: tests/commands/testOpen.py:95
#, python-format
msgid "ITEMS_IN_CONTAINER_%s"
msgstr ""

#: core/commands/take.py:15 tests/commands/testTake.py:48
#: tests/commands/testTake.py:53 tests/commands/testTake.py:58
#: tests/commands/testTake.py:63
msgid "ERROR_TAKE_NO_ITEM_GIVEN"
msgstr ""

#: core/commands/take.py:17 tests/commands/testTake.py:27
#: tests/commands/testTake.py:32
msgid "ERROR_TAKE_TOO_LOW_QUANTITY"
msgstr ""

#: core/commands/take.py:19 tests/commands/testTake.py:17
#: tests/commands/testTake.py:22
msgid "ERROR_TAKE_INVALID_FORMAT_QUANTITY"
msgstr ""

#: core/commands/take.py:21 tests/commands/testTake.py:120
#: tests/commands/testTake.py:125
msgid "ERROR_TAKE_INVALID_CONTAINER_INDEX"
msgstr ""

#: core/commands/take.py:27 tests/commands/testTake.py:68
#: tests/commands/testTake.py:73
msgid "ERROR_TAKE_UNKNOWN_ITEM"
msgstr ""

#: core/commands/take.py:43 tests/commands/testTake.py:78
#: tests/commands/testTake.py:83
msgid "ERROR_TAKE_ITEM_NOT_AVAILABLE"
msgstr ""

#: core/commands/take.py:46 tests/commands/testTake.py:37
#: tests/commands/testTake.py:42
#, python-format
msgid "ERROR_TAKE_QUANTITY_TOO_HIGH_%s"
msgstr ""

#: core/commands/take.py:59 tests/commands/testTake.py:149
#: tests/commands/testTake.py:159 tests/commands/testTake.py:169
#: tests/commands/testTake.py:179 tests/commands/testTake.py:189
#: tests/commands/testTake.py:199
#, python-format
msgid "TAKE_CONFIRMATION_%(quantity)s_%(name)s"
msgstr ""

#: core/commands/talk.py:18 tests/commands/testTalk.py:16
#: tests/commands/testTalk.py:21
msgid "ERROR_TALK_NO_CHARACTER_GIVEN"
msgstr ""

#: core/commands/talk.py:20 tests/commands/testTalk.py:26
#: tests/commands/testTalk.py:31
msgid "ERROR_TALK_NO_SENTENCE_GIVEN"
msgstr ""

#: core/commands/talk.py:29 tests/commands/testTalk.py:36
#: tests/commands/testTalk.py:41
msgid "ERROR_TALK_UNKNOWN_CHARACTER"
msgstr ""

#: core/commands/talk.py:36 tests/commands/testTalk.py:46
msgid "ERROR_TALK_UNKNOWN_SENTENCE"
msgstr ""

#: models/area.py:21 tests/commands/testLook.py:24
#: tests/commands/testLook.py:42 tests/commands/testLook.py:86
#: tests/commands/testLook.py:92 tests/commands/testMove.py:44
#: tests/commands/testMove.py:46 tests/commands/testMove.py:49
#: tests/commands/testMove.py:51
msgid "DIRECTION_KEY_NORTH"
msgstr ""

#: models/area.py:22 tests/commands/testMove.py:34
#: tests/commands/testMove.py:39
msgid "DIRECTION_KEY_EAST"
msgstr ""

#: models/area.py:23
msgid "DIRECTION_KEY_SOUTH"
msgstr ""

#: models/area.py:24
msgid "DIRECTION_KEY_WEST"
msgstr ""

#: models/area.py:39
msgid "AREA_TYPE_DUNGEON"
msgstr ""

#: models/area.py:39
msgid "AREA_TYPE_LAND"
msgstr ""

#: models/item_container.py:32 tests/commands/testDrop.py:95
#: tests/commands/testDrop.py:102 tests/commands/testOpen.py:36
#: tests/commands/testOpen.py:41 tests/commands/testTake.py:89
#: tests/commands/testTake.py:94
msgid "ERROR_UNKNOWN_ITEM_CONTAINER_TYPE_LABEL"
msgstr ""

#: models/place.py:15 models/place.py:29 models/place.py:39 models/place.py:49
#: tests/commands/testEnter.py:24 tests/commands/testExit.py:23
#: tests/commands/testExit.py:28
msgid "PLACE_TYPE_CAVE"
msgstr ""

#: models/place.py:15 models/place.py:27 models/place.py:37 models/place.py:47
#: tests/commands/testEnter.py:44 tests/commands/testEnter.py:46
#: tests/commands/testEnter.py:51 tests/commands/testEnter.py:53
#: tests/commands/testEnter.py:58 tests/commands/testEnter.py:63
#: tests/commands/testExit.py:43 tests/commands/testExit.py:48
#: tests/commands/testExit.py:53 tests/commands/testExit.py:55
#: tests/commands/testExit.py:60 tests/commands/testExit.py:62
msgid "PLACE_TYPE_DUNGEON"
msgstr ""

#: models/place.py:25 models/place.py:35 models/place.py:45
#: tests/commands/testEnter.py:36 tests/commands/testEnter.py:41
#: tests/commands/testExit.py:35 tests/commands/testExit.py:40
msgid "ERROR_UNKNOWN_PLACE_TYPE"
msgstr ""

#: models/place.py:96
msgid "ERROR_PLACE_GENERATION"
msgstr ""

#: models/player.py:33
msgid "ERROR_CONNECT_INVALID_CREDENTIALS"
msgstr ""
